src/routes/api/users.js

```diff
@@ -0,0 +1,79 @@
+import express from "express";
+import knex from "../../db/index";
+
+
+// GET ALL
+// http://localhost:8989/api/user/all METHOD = GET
+const users = express.Router();
+
+users.get("/all", function(req, res) {
+    knex
+      .select()
+      .from("Event")
+      .then(data => {
+        res
+          .status(200)
+          .send(data)
+          .end();
+      })
+      .catch(error => {
+        res
+          .status(500)
+          .send("Database error: " + error.errno)
+          .end();
+      });
+  });
+
```

Comment:
I think, only one empty line is enough for separating different API services. No need to add many extra lines